File: src/main/java/io/github/alexiscomete/lapinousecond/WorkEnum.java
package io.github.alexiscomete.lapinousecond;

public enum WorkEnum {

    WORK1("default", "*Vous aidez quelqu'un à trouver le salon 'memes', vous gagnez rc Rabbitcoins !*", 15, 35),
    WORK2("default", "*Vous aider un bot à se rappeler de son préfix, vous gagnez rc Rabbitcoins !*", 16, 45),
    WORK3("default", "*Vous trouvez billet de train pour un autre serveur que vous rendez à son propriétaire, il vous récompense avec rc Rabbitcoins ...*", 15, 36),
    WORK4("default", "*Quelqu'un vous parle de Lapinou Premier, il vous donne rc Rabbitcoins car vous l'aider à le chercher, bien sûr personne ne le trouve.*", 15, 38),
    WORK5("default", "*Le serveur vous récompense avec rc Rabbitcoins pour avoir défendu le serveur contre Alphabet Lapin !*", 15, 55),
    WORK6("default", "*Carlos le chat virtuel est passé par là ... vous gagnez rc Rabbitcoins ...*", 15, 40),
    WORK7("default", "*Vous passez un peu de temps avec les Togerts, ils vous donnent rc Rabbitcoins*", 15, 40),
    WORK8("default", "*Vous échangez une information sur le Wumpus d'or contre un peu d'argent (rc)", 25, 70),
    WORK9("video games", "*Vous écrivez un article sur Minecraft qui vous rapporte rc Rabbitcoins ...*", 30, 80),
    WORK10("computer science", "*Vous recodez le système informatique en C++, on vous récompense avec rc Rabbitcoins", 40, 90);

    String type;
    String answer;
    int min;
    int max;

    WorkEnum(String type, String answer, int min, int max) {
        this.type = type;
        this.answer = answer;
        this.min = min;
        this.max = max;
    }
}

File: src/main/java/io/github/alexiscomete/lapinousecond/entity/concrete/resources/WorkEnum.kt
package io.github.alexiscomete.lapinousecond.entity.concrete.resources

enum class WorkEnum(
    val type: String,
    val answer: String,
    var min: Int,
    var max: Int,
    val resource: Resource?,
    var coef: Int
) {
    WORK1(
        "default",
        "*Vous aidez quelqu'un à trouver le salon 'memes', vous gagnez rc ${Resource.RABBIT_COIN.show} !*",
        15,
        35,
        null,
        2
    ),
    WORK2(
        "default",
        "*Vous aider un bot à se rappeler de son préfix, vous gagnez rc ${Resource.RABBIT_COIN.show} !*",
        16,
        45,
        null,
        2
    ),
    WORK3(
        "default",
        "*Vous trouvez billet de train pour un autre serveur que vous rendez à son propriétaire, il vous récompense avec rc ${Resource.RABBIT_COIN.show} ...*",
        15,
        36,
        null,
        2
    ),
    WOOD1(
        "default",
        "Vous coupez du bois dans une forêt proche. Vous revenez avec rc ${Resource.WOOD.show}",
        100,
        200,
        Resource.WOOD,
        20
    ),
    STONE1(
        "default",
        "Vous allez récupérer rc ${Resource.STONE.show} sur une colline",
        80,
        150,
        Resource.STONE,
        20
    ),
    BRANCH1(
        "default",
        "Vous récupérez rc ${Resource.BRANCH.show} mortes dans la forêt",
        50,
        250,
        Resource.BRANCH,
        15
    ),
    BRANCH2("default", "Vous cassez rc ${Resource.BRANCH.show} sur des arbres.", 150, 250, Resource.BRANCH, 15), DIAMOND1(
        "default",
        "En allant minez vous trouvez rc diamant(s) ...",
        1,
        10,
        Resource.DIAMOND,
        1
    );

}